src/picasso-definition/index.js

```diff
@@ -41,6 +42,15 @@ export default function createPicassoDefinition({
 
   const componentDefinitions = [...components, ...selectables.components];
 
+  // Disable animations if there are too many points
+  if (layoutService.getHyperCubeValue('qSize.qcy', 0) > NUMBERS.MAX_NR_ANIMATION) {
```

Comment:
I think it would be nicer to move this logic into the viewHandler, and have a function/get property in there which we call `animationsEnabled`. Then we use that one to decide wether or not to enable animations in the component definitions:

```
animations: {
      enabled: viewHandler.animationsEnabled,
```

Reply:
good point!